components/KeyFigures.vue

```diff
@@ -27,38 +27,20 @@
 <!-- NOT scoped so we can be inherited by other components which use Figures -->
 <style lang="scss">
   .figures {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
```

Comment:
CSS Grid used to be wrapped in `@supports` which made it much safer to apply other properties but IE11 doesn't actually render flexbox inside a `@supports` block. yay!